dashboard/utils.py

```diff
@@ -466,44 +465,19 @@ def get_exam_card_status(self):  # pylint: disable=too-many-return-statements
 
         user = self.user
         try:
-            exam_profile = ExamProfile.objects.only('status').get(profile=user.profile)
+            ExamProfile.objects.only('status').get(profile=user.profile)
         except ExamProfile.DoesNotExist:
             return ExamProfile.PROFILE_ABSENT
-        if settings.FEATURES.get('ENABLE_EDX_EXAMS', False):
-            auths = ExamAuthorization.objects.filter(
-                user=user,
-                status=ExamAuthorization.STATUS_SUCCESS,
-                exam_run__in=future_runs,
-            )
-            if auths.exists():
-                return ExamProfile.PROFILE_SCHEDULABLE
-            else:
-                return ExamProfile.PROFILE_SUCCESS
-
-        if exam_profile.status in (ExamProfile.PROFILE_PENDING, ExamProfile.PROFILE_IN_PROGRESS,):
-            return ExamProfile.PROFILE_IN_PROGRESS
-
-        elif exam_profile.status in (ExamProfile.PROFILE_INVALID, ExamProfile.PROFILE_FAILED,):
-            return ExamProfile.PROFILE_INVALID
```

Comment:
Don't we need that checks anymore? Is that intentional?

Reply:
The various states of the exam profile are not used anymore because we are not syncing ExamProfiles with Pearson.

Reply:
okay, then we are good to go.